### PR TITLE
Update Librosa Version To V0.10.0

### PR DESCRIPTION
**Pull request by MattyB95**
Update Librosa Version To V0.10.0

### CODE REVIEW

**Comment by erogol** [comment]
Tx for the PR. Is it just convenience or had a problem that you'd need to update librosa?

**Comment by MattyB95** [comment]
Hi @erogol, purely convenience. I was reviewing the change log for Librosa, and this version mentioned speedups and enhancements. I also did want to start working with this repository and wanted to begin making small additions to an open-source project :)

> This release provides many speedups and enhancements to core operations, faster importing and loading, and full type annotations.

File: requirements.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ scipy>=1.4.0
 torch>=1.7
 torchaudio
 soundfile
-librosa==0.8.0
+librosa>=0.10.0
```
Comment (erogol) [inline requirements.txt:9]: we need to bind it to ==0.10.0 to prevent incompat issues in the future.

**Review by erogol — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (MattyB95) [reply requirements.txt:9]: Yep, I agree. Would you be happy with `librosa==0.10.0.*` as there is technically a post1 and post2 version available?

**Comment by erogol** [comment]
thanks for the PR!

**Pull request merged**